docs-source/content/security/certificates.md

```diff
@@ -24,14 +24,31 @@ certificate(s) and key:
 ```bash
 $ kubectl create secret tls weblogic-operator-cert -n weblogic-operator-ns \
   --cert=<path-to-certificate> --key=<path-to-private-key>
+```
+
+For Helm 2.x:
 
+```bash
 $ helm get values weblogic-operator
 
 $ helm upgrade --wait --recreate-pods --reuse-values \
   --set externalRestEnabled=true \
   --set externalRestIdentitySecret=weblogic-operator-cert \
   weblogic-operator kubernetes/charts/weblogic-operator
 ```
+
+For Helm 3.x:
+
+```bash
+$ helm get values weblogic-operator -n weblogic-operator-ns
+
+$ helm -n weblogic-operator-ns upgrade weblogic-operator kubernetes/charts/weblogic-operator \
+  --wait --recreate-pods --reuse-values \
+  --set externalRestEnabled=true \
+  --set externalRestIdentitySecret=weblogic-operator-cert 
+```
+
+
```

Comment:
minor:  can you modify the 2.x argument order so that they line-up with 3.x and make it obvious what the difference is?  I assume 2.x could look exactly like 3.x except that it won't have the '-n' clause.

Reply:
I will adjust that.  Hem 2.x can have --namespace esp if multiple ns are  involved but it can't take -n.   Helm 3.x on the other hand takes -n and not --namespace.    .